Cell actions being added to more places in Security app (#3296)
elastic/security-docs: docs/dashboards/entity-dashboard.asciidoc

* Select the *Host risk classification* menu to filter the chart by the selected classification. 
* Click a host name link to go to the Host details page.
* Hover over a host name link to display inline actions: *Add to timeline*, which adds the selected value to a new Timeline, and *Copy to Clipboard*, which copies the host name value for you to paste later. 
* Click *View all* in the upper-right to display all host risk information on the Hosts page. 

> * Click the number link in the *Alerts* column to view the alerts on the Alerts page. Hover over the number and select *Investigate in timeline* (image:images/timeline-button-osquery.png[Investigate in timeline icon,20,20]) to launch Timeline, which populates a query with the selected host name value.

Reviewer: ```suggestion
* Click the number link in the *Alerts* column to view the alerts on the Alerts page. Hover over the number and select *Investigate in timeline* (image:images/timeline-button-osquery.png[Investigate in timeline icon,20,20]) to launch Timeline with a query that includes the associated host name value.
```
Suggested "associated: rather than "selected" since technically the user selected alerts associated with a host. If you decide to include this suggestion, it applies below on the same page in the "users" section as well.

* Select the *User risk classification* menu to filter the chart by the selected classification. 
* Click a user name link to go to the User details page. 
* Hover over a host name link to display inline actions: *Add to timeline*, which adds the selected value to a new Timeline, and *Copy to Clipboard*, which copies the user name value for you to paste later. 
* Click *View all* in the upper-right to display all user risk information on the Users page. 

> * Click the number link in the *Alerts* column to view the alerts on the Alerts page. Hover over the number and select *Investigate in timeline* (image:images/timeline-button-osquery.png[Investigate in timeline icon,20,20]) to launch Timeline, which populates a query with the selected user name value.

Author: ```suggestion
* Click the number link in the *Alerts* column to view the alerts on the Alerts page. Hover over the number and select *Investigate in timeline* (image:images/timeline-button-osquery.png[Investigate in timeline icon,20,20]) to launch Timeline with a query that includes the associated user name value.
```
